Open Judge System/Workers/OJS.Workers.ExecutionStrategies/NodeJsZipExecuteHtmlAndCssStrategy.cs

```diff
@@ -0,0 +1,325 @@
+﻿namespace OJS.Workers.ExecutionStrategies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text.RegularExpressions;
+    using Checkers;
+    using Common;
+    using Executors;
+    using Ionic.Zip;
+    using OJS.Common;
+    using OJS.Common.Extensions;
+
+    public class NodeJsZipExecuteHtmlAndCssStrategy : NodeJsPreprocessExecuteAndRunUnitTestsWithMochaExecutionStrategy
+    {
+        protected const string EntryFileName = "*.html";
+        protected const string SubmissionFileName = "_$submission";
+
+        public NodeJsZipExecuteHtmlAndCssStrategy(
+           string nodeJsExecutablePath,
```

Comment:
all params must be 2 tabs from row 19; needs confirmation from @KristianMariyanov